src/obb/models/game_info.cljc

```diff
@@ -1,23 +1,32 @@
 (ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
   obb.models.game-info
-  "This model represents a game between two players")
+  "This model represents a game between two players"
+  (:require [schema.core :as s]))
 
-(defn create
+(def IdSchema s/Str)
+
+(def GameInfo
```

Comment:
This is again a detail, but we are using the suffix `Info` for something that is really the data for a `Game`. Will we use the same policy for all the other entities and have a `PlayerInfo` and a `UnitInfo`? I know I complain too much about naming, but I think it is specially important to pick the right ones in the beginning.

Ignore if you just think this is not relevant.